### ISSUE TITLE
Live-collaboration has strange participants 

### ISSUE THREAD

**Issue by AaronRobinson** [excalidraw/excalidraw#2780]
I keep seeing more cursors than people I’ve shared live-collab links with. They join, the cursor moves around a bit then drops off. The name is my name. Others are seeing the same issue. I’m not using Excalidraw elsewhere. Is there any way there could be a lag with the collab events?

**Comment by ecolss** [comment]
Same issue encountered on my macbook, Chrome.

**Comment by dwelle** [comment]
Yes, this is definitely an issue. I've been only able to consistently reproduce locally, doing something like quickly refreshing the browser and stuff.

**Comment by gre** [comment]
same issue. a few cursors appears and started selecting random things. it was very spooky 🤣

**Comment by dwelle** [comment]
Don't worry, it definitely wasn't any 3rd party :). Nevertheless, I agree that it's not nice. We'll try to fix this soon.

**Comment by JonathanHarvey** [comment]
I've experienced this issue as well. Seeing up to 5 additional users shown and a single delayed replication of my movements presented back to me. Experienced in Chrome. It's unfortunately very distracting, especially when things get left selected by one of these phantom users.
